common/src/main/scala/com/gu/recipeasy/models/CuratedRecipe.scala

```diff
@@ -129,7 +130,7 @@ object CuratedRecipe {
       ),
       published = Some(
         ChangeRecord(
-          date = r.publicationDate.toInstant.toEpochMilli,
+          date = DateTime.now.toInstant.getMillis,
```

Comment:
I vote for having the atom behave independently to the article. Mostly because one day we may have a atoms which do not have corresponding articles and in this case the coupling would not make sense. 

But others may disagree :)